### ISSUE TITLE
uk-tooltip not showing when hovering over icon in input control

### ISSUE THREAD

**Issue by arnvanhoutte** [uikit/uikit#3194]
### UIkit version
3.0.0-beta.39

### Browser
Firefox

### Reproduction Link
https://codepen.io/anon/pen/KQVaPa

### Steps to reproduce
Hover over input icon

### What is Expected?
Tooltip appears

### What is actually happening?
Tooltip does not appear

**Comment by aarongerig** [comment]
You have to define a title (either an dedicated attribute on the same element or inside the `uk-tooltip` attribute). Have another look at the documentation: https://getuikit.com/docs/tooltip

https://codepen.io/aarongerig/pen/EQPWaB

Does this resolve your question? If so, please close this issue.

**Comment by arnvanhoutte** [comment]
My apologies, but I forgot to save the pen after I edited it. Now it's the correct one

**Comment by aarongerig** [comment]
Yeah, it‘s because of that line: https://github.com/uikit/uikit/blob/develop/src/less/components/form.less#L684

Why don‘t you set the tooltip on the input?

**Comment by arnvanhoutte** [comment]
>Why don‘t you set the tooltip on the input?

Because it only should show up when hovering over the icon (to not disturb the users too much)
